server/src/main/java/PerkManagerController.java

```diff
@@ -0,0 +1,26 @@
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+//import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/perks")
+@CrossOrigin(origins = "http://localhost:5173")
```

Comment:
are you keeping this in for testing right now? I think youre gonna have to remove the localhost later since its being deployed

Reply:
yeah sure thats fine for now then